### DISCUSSION TITLE
Background is slightly darker than the terminal color theme (Catppuccin Latte)

### DISCUSSION THREAD

**Discussion by ChristianJacobsen** [LazyVim/LazyVim#1225] [Q&A] [answered]
Hi!

I'm struggling to match my terminal background and Neovim's background when using LazyVim.
I use the Catppuccin Latte theme, but it is slightly darker in Neovim.

![image](https://github.com/LazyVim/LazyVim/assets/511624/e7246e61-fb38-4013-ba0d-d17a9b76cf2f)

As you can see from the image, my terminal padding is slightly lighter than the Neovim background.
Does LazyVim further modify the color scheme in any way?

The only thing I've done with theming is add the Catppuccin as a plugin:
```lua
return {
    {
        'catppuccin/nvim',
        name = 'catppuccin',
        config = function()
            require('catppuccin').setup({
                flavour = 'latte',
            })
        end,
    },
}
```

Any suggestions as to how I fix this are greatly appreciated 😄

**Comment by folke** [comment]
not lazyvim

**Reply by ChristianJacobsen** [reply]
I hear you say that, but I get the correct background color when using plain Neovim with Lazy loading just Catppuccin.

```lua
-- $ cat .config/nvim/init.lua
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "https://github.com/folke/lazy.nvim.git",
    "--branch=stable",
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)

require("lazy").setup({
  {
    "catppuccin/nvim",
    name = "catppuccin",
    config = function()
      require("catppuccin").setup({
        flavour = "latte",
      })
    end,
  },
})
```

That's why I asked if LazyVim('s dependencies) does something additional to the background color. This is also not the case with AstroNvim, which I used before trying out LazyVim.

I'm not asking you to solve this for me. Only if you knew about any background color manipulation going on.

I will continue looking.

**Reply by ChristianJacobsen** [reply, marked as answer]
I had just forgotten to tell LazyVim to use Catppuccin as a theme through the `opts` variable, so it was still using `tokyonight`.

Thanks anyways 😄
